### PR TITLE
Feature/routing service

### PR DESCRIPTION
**Pull request by CloudNiner**
This gets a basic map drawing, centered on the config bounds/center and zoomed to the geojson routing response. 

Deferred Things:
- Tests
- Styling the geojson/markers based on their properties
- Adding destination param to the routing view url so it can be passed to the request
- Getting origin param from browser location

### CODE REVIEW

File: src/nih_wayfinding/app/scripts/config.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+(function () {
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/app/scripts/config.js:2]: yay configs!

File: src/nih_wayfinding/app/scripts/mapping/map-service.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    'use strict';
+
+    /* ngInject */
+    function Map (leafletBoundsHelpers, Config, MapStyle) {
+
+        var defaultBounds = Config.bounds;
+        var defaultCenter = Config.center;
+        var defaultLegend = {
+            colors: MapStyle.getLegendRamp(),
+            labels: ['Recent Review', 'Older Review', 'Ancient Review']
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/app/scripts/mapping/map-service.js]: Hm, maybe 'ancient' isn't the best label.  Guess we can change that later.

**Comment by flibbertigibbet** [comment]
+1

**Comment by flibbertigibbet** [comment]
Looks like Travis had some weird npm error; might want to restart it or run tests locally before merging.

**Comment by CloudNiner** [comment]
Hmm I saw this error yesterday, but assumed it was npm being npm. Now I'm not so sure, going to investigate.

Tests pass locally, so I'm going to merge.

**Comment by CloudNiner** [comment]
Aaaaand it's just npm being itself.

**Pull request merged**